tox.ini

```diff
@@ -203,6 +204,8 @@ deps =
     adapter_waitress-waitress02: waitress<2.1
     adapter_waitress-waitresslatest: waitress
     agent_features: beautifulsoup4
+    agent_features-{py37,py38,py39,py310,py311,pypy37}: protobuf
```

Comment:
We should probably have some tests that run without protobuf as well to make sure that code path works as well.

Reply:
I was trying to think of how to make that work. Possibly have a debug setting that forces JSON serialization?

Reply:
Yeah you might be able to do that. It's a little tricky cause it's at import time inside of the nr core but yes perhaps just a switch would be good enough. The disadvantage of doing that is you are mocking a little bit so you would miss validating the exception logic around protobuf imports failing. I was thinking maybe just run a couple test envs with protobuf and the rest without. Something like:
```
python-agent_features-{py27,py311}-with_protobuf-without_extensions,
.
.
.
agent_features-{py311}-with_protobuf: protobuf
```